__tests__/__resources__/properties/default_properties.yaml

```diff
@@ -97,7 +97,7 @@ provisioning:
 workflows:
   # The system where you want to run your workflow
   # (must exist and be active within your zOSMF)
-  system: system
+  system: workflow-system
```

Comment:
I don't see a problem with this.
🤔 How about we set some sort of standard?
Each value will have a "full path" of its property
For example: `zosjobs-iefbr14Member` will be the value of `zosjobs.iefbr14Member`

Reply:
That way, they will always be unique and traceable (hopefully making it easier for automation purposes

Reply:
Sounds reasonable. Are you suggesting this change be made in this PR?

Reply:
I guess 😝